app/models/container_quota_scope.rb

```diff
@@ -0,0 +1,3 @@
+class ContainerQuotaScope < ApplicationRecord
+  belongs_to :container_quota
```

Comment:
what does this class do ?
why we need this class ?

Reply:
from discussion in https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/190 is sounds like this class only hold one string field (`:scope` [string] the name of the scope ) that can currently hold only 4 possible strings ... 
Do we need a class to hold a list of pre defined strings, shouldn't we use a text field here ?

Reply:
This is a model class.  We need it to access the table, because Rails ;-)
As we're discussing on that PR:
1. schema might be suboptimal, but gaprindashvili schema is frozen and we need this feature there.
2. since we're going to retain full quotas history by archiving instead of in-place updates, there is a case where this schema is *more* effecient to refresh.

Reply:
@cben does a quata has only 4 possible set of scopes ?
a. terminating and best effort
b. terminating and not best effort
c. not terminating and best effort
d not terminating and not best effort